genienlp/server.py

```diff
@@ -186,6 +222,7 @@ def parse_argv(parser):
                         help='Checkpoint file to use (relative to --path, defaults to best.pth)')
     parser.add_argument('--port', default=8401, type=int, help='TCP port to listen on')
     parser.add_argument('--stdin', action='store_true', help='Interact on stdin/stdout instead of TCP')
+    parser.add_argument('--database_dir', type=str, help='Database folder containing all relevant files')
```

Comment:
The database needs to be stored in the same folder as the model, because there is no way to specify a different path to the server.

Reply:
But the database is independent of the model. Aren't we passing an ENV variable for the database?